content/docs/cve/cveTeams.md

```diff
@@ -0,0 +1,55 @@
+---
+# Title, summary, and page position.
+linktitle: Teams: Session Management
```

Comment:
This causes an error (You need to put this inside quotes I think). See the error logs where it says:
```
12:25:24 PM: Error: Error building site: "/opt/build/repo/content/docs/cve/cveTeams.md:3:1": failed to unmarshal YAML: yaml: line 2: mapping values are not allowed in this context
```

```suggestion
linktitle: "Teams: Session Management"
```